kas/colibri-imx6ull.yml

```diff
@@ -0,0 +1,22 @@
+header:
+  version: 11
+  includes:
+  - kas/include/mender-base.yml
+  - kas/include/nxp.yml
+  - kas/include/toradex.yml
+
+machine: colibri-imx6ull
+
+local_conf_header:
+  colibri-imx8x: |
+    INHERIT:remove = " mender-full "
+    INHERIT:append = " mender-full-ubi "
```

Comment:
Although I realize it will add a lot of lines, is it perhaps better to include `INHERIT:append = " mender-full"` in all regular boards instead of assuming it's there already, and removing it here? It might be more self contained, in a sense. `remove` is often a sign that you have added too much to the default setup.